apps/heft/src/stages/BuildStage.ts

```diff
@@ -153,20 +161,32 @@ export interface ICompileSubstageProperties {
 /**
  * @public
  */
-export interface IBundleSubstageProperties {
+export interface ISharedBundleSubstageWebpackProperties {
   /**
    * A path to a Webpack configuration JS file. If this isn't specified, and a Webpack
    * configuration isn't specified via another plugin, Webpack won't be run.
    */
   webpackConfigFilePath?: string;
 
+  /**
+   * A path to a Webpack configuration JS file, specifically to be used in --serve mode. If this and
+   * webpackConfigFilePath aren't specified, and a Webpack configuration isn't specified
+   * via another plugin, Webpack won't be run in --serve mode.
```

Comment:
Is `--serve`  a Heft parameter?

Reply:
Oops. The dashes shouldn't be there.

Reply:
From our conversation, I'm proposing to this feature `heft dev-server` (instead of `webpack --serve`). The interface would be like this:

**package.json**
```js
{
  . . .
  "scripts": {
    "start": "heft dev-server"
  }
}
```